slackbot_deploy.yml

```diff
@@ -0,0 +1,102 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: slackbot
+  labels:
+    app: slackbot
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: customer1bot
+  namespace: slackbot
+  labels:
+    bot: customer1bot
+    app: slackbot
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      bot: customer1bot
+  template:
+    metadata:
+      labels:
+        bot: customer1bot
+        app: slackbot
+    spec:
+      containers:
+        - name: bot
+          image: slackbot
+          env:
+            - name: CUSTOMER_NAME
+              valueFrom:
+                secretKeyRef:
+                  name: customer1bot-config
+                  key: CUSTOMER_NAME
+            - name: SLACK_PAGERDUTY_DIRECTORY
+              valueFrom:
+                secretKeyRef:
+                  name: customer1bot-config
+                  key: PAGERDUTY_DIRECTORY
+            - name: SLACK_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: customer1bot-config
+                  key: SLACK_TOKEN
+            - name: SLACK_BOT_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: customer1bot-config
+                  key: SLACK_BOT_USERNAME
+            - name: PAGERDUTY_LINK
+              valueFrom:
+                secretKeyRef:
+                  name: customer1bot-config
+                  key: PAGERDUTY_LINK
+            - name: PAGERDUTY_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: customer1bot-config
+                  key: PAGERDUTY_API_KEY
+            - name: PAGERDUTY_FROM_EMAIL
+              valueFrom:
+                secretKeyRef:
+                  name: customer1bot-config
+                  key: PAGERDUTY_FROM_EMAIL
+            - name: PAGERDUTY_SERVICE_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: customer1bot-config
+                  key: PAGERDUTY_SERVICE_API_KEY
+            - name: PAGERDUTY_SUPPORT_SCHEDULE
+              valueFrom:
+                secretKeyRef:
+                  name: customer1bot-config
+                  key: PAGERDUTY_SUPPORT_SCHEDULE
+            - name: PAGERDUTY_SUPPORT_SERVICE
+              valueFrom:
+                secretKeyRef:
+                  name: customer1bot-config
+                  key: PAGERDUTY_SUPPORT_SERVICE
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: customer1bot-config
+  namespace: slackbot
+  labels:
+    bot: customer1bot
+    app: slackbot
+type: Opaque
+stringData:
+  CUSTOMER_NAME: "Awesome Customer Brand"
```

Comment:
Why not simply `Brand Name` or `Acme Corp`?

Reply:
cause I'm silly? :)

Changed to a more serious one.